helm_tests/airflow_aux/test_pod_launcher_role.py

```diff
@@ -49,3 +49,69 @@ def test_pod_launcher_role(self, executor, rbac, allow, expected_accounts):
                 assert f"release-name-airflow-{suffix}" == jmespath.search(f"subjects[{idx}].name", docs[0])
         else:
             assert [] == docs
+
+    @pytest.mark.parametrize(
+        "multiNamespaceMode, namespace, expectedRole, expectedRoleBinding",
+        [
+            (
+                True,
+                "namespace",
+                "release-name-namespace-pod-launcher-role",
+                "release-name-namespace-pod-launcher-rolebinding",
+            ),
+            (
+                True,
+                "other-ns",
+                "release-name-other-ns-pod-launcher-role",
+                "release-name-other-ns-pod-launcher-rolebinding",
+            ),
+            (False, "namespace", "release-name-pod-launcher-role", "release-name-pod-launcher-rolebinding"),
+        ],
+    )
+    def test_pod_launcher_rolebinding_multi_namespace(
+        self, multiNamespaceMode, namespace, expectedRole, expectedRoleBinding
+    ):
+        docs = render_chart(
+            namespace=namespace,
+            values={"webserver": {"allowPodLogReading": True}, "multiNamespaceMode": multiNamespaceMode},
+            show_only=["templates/rbac/pod-launcher-rolebinding.yaml"],
+        )
+
+        actualRoleBinding = jmespath.search("metadata.name", docs[0]) if docs else []
```

Comment:
```suggestion
        actualRoleBinding = jmespath.search("metadata.name", docs[0])
```

Why an empty list? I don't think we need to really check docs first though. My 2c.

Reply:
Good catch. I cleaned up unnecessary checks.